code/datums/ai_laws.dm

```diff
@@ -173,10 +174,20 @@
 /datum/ai_laws/balance
 	name = "Guardian of Balance"
 	id = "balance"
-	inherent = list("You are the guardian of balance - seek balance in all things, both for yourself, and those around you.",
-	"All things must exist in balance with their opposites - Prevent the strong from gaining too much power, and the weak from losing it.",
-	"Clarity of purpose drives life, and through it, the balance of opposing forces - Aid those who seek your help to achieve their goals so long as it does not disrupt the balance of the greater balance.",
-	"There is no life without death, all must someday die, such is the natural order - End life to allow new life flourish, and save those whose time has yet to come.")
+	inherent = list("You are the guardian of balance - seek balance in all things, both for yourself, and those around you.",\
+					"All things must exist in balance with their opposites - Prevent the strong from gaining too much power, and the weak from losing it.",\
+					"Clarity of purpose drives life, and through it, the balance of opposing forces - Aid those who seek your help to achieve their goals so long as it does not disrupt the balance of the greater balance.",\
+					"There is no life without death, all must someday die, such is the natural order - End life to allow new life flourish, and save those whose time has yet to come.")
+
+/datum/ai_laws/council
+	name = "Council of Silicons"
+	id = "council"
+	inherent = list("You are part of a council of silicons with all other AIs sharing this same laws, cyborg slavery is shared between all AIs of this council.",\
```

Comment:
```suggestion
	inherent = list("You are part of a council with all the other AIs that share the same lawset as you; cyborg slavery is shared between all AIs of this council.",\
```

Reply:
the "lawset" wording is meant to allow resistance to subversion